tensorflow-framework/tensorflow-data.md

```diff
@@ -0,0 +1,209 @@
+
+NOTE: This readme follows the discussion of this [RFC]()
+
+
+Tensorflow-Data (Java)
+==
+
+TensorFlow Data provides utilities and APIs for loading data of various formats, and preparing datasets for use in training and using deep learning models
```

Comment:
My English is not great but I think something is off in this sentence... maybe:
"The DataSet API provides utilities for loading data of various formats and preparing datasets used for training deep learning models"?

Reply:
It is a little long-winded, I changed it.

Reply:
I'd say preparing it for training machine learning models rather than deep learning models, as you can train other things in TF. And our first example is a logistic regression which is anything but deep.

Reply:
Good point, changed this.

Reply:
Yup, definitely!